### ISSUE TITLE
Spaces in folder name

### ISSUE THREAD

**Issue by adamo** [OFFLINE-GmbH/oc-responsive-images-plugin#23]
Quick info that when a folder has spaces the plugin produces a error in chrome 

`Failed parsing 'srcset' attribute value since it has an unknown descriptor`

I just changed the code as follow but maybe you guys have a different approach

```
/**
     * Generates the public url for a path
     *
     * @param $path
     *
     * @return mixed
     */
    protected function getPublicUrl($path)
    {
        $relativePath = str_replace(base_path(), '', $path);
        $filename = basename($relativePath);
        $relativeFolderPath = str_replace($filename, '', $relativePath);

        return URL::to('/') . str_replace(' ', '%20',$relativeFolderPath) . rawurlencode($filename) ;
    }
```

**Comment by tobias-kuendig** [comment]
Thank you for the bug report! This has been fixed in b02cd6325a3fefb85d14340d104be01c0c2d9627 and released as 1.1.9.

**Comment by adamo** [comment]
Your welcome, glad I could help.
